DiscoveryV2.playground/Pages/Training data.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,76 @@
+//:## Training data
+
+import DiscoveryV2
+
+let discovery = setupDiscoveryV2()
+let documentID = "<your-document-id>"
```

Comment:
if these variables are showing up in the API ref, and if we're not using "real" IDs in the examples, they need to follow the snake_case pattern `{document_id}`